circuit_knitting_toolbox/circuit_cutting/__init__.py

```diff
@@ -14,6 +14,51 @@
 
 .. currentmodule:: circuit_knitting_toolbox.circuit_cutting
 
+Circuit Cutting
+===============
+
+.. autosummary::
+    :toctree: ../stubs/
+    :nosignatures:
+
+    partition_circuit_qubits
+    partition_problem
+    decompose_gates
+    execute_experiments
+    reconstruct_expectation_values
+
+.. autosummary::
+    :toctree: ../stubs/
+    :nosignatures:
+    :template: autosummary/class_no_inherited_members.rst
+
+    PartitionedCuttingProblem
```

Comment:
These named tuples need to be included in the API docs because they are referenced as return types in our cutting functions